bigquery-writer/bigquery-writer.py

```diff
@@ -97,38 +160,37 @@ def start(self):
             sleep(1000)
             return
```

Comment:
I don't really see the need to consolidate them in one function -- I don't believe in creating new functions just to keep functions below an artificial length. I believe that makes the code harder to read in the end.